user-service/src/main/java/uk/gov/gchq/palisade/service/user/web/UserController.java

```diff
@@ -60,22 +57,11 @@ public Boolean addUserRequest(@RequestBody final AddUserRequest request) {
         return service.addUser(request.user).equals(request.user);
     }
 
-    @PostConstruct
+    @EventListener(ApplicationReadyEvent.class)
```

Comment:
Pros and cons vs PostConstruct? Readiness of spring beans and autowiring stuff i presume?

Reply:
Turns out that methods that have the @Chacheable annotation have no effect when used with an @PostConstruct annotated method.

Reply:
I sense this one may have driven you mad for a while...